clang/test/CodeGenCXX/amdgpu-buffer-rsrc-typeinfo.cpp

```diff
@@ -0,0 +1,9 @@
+// REQUIRES: amdgpu-registered-target
+// RUN: %clang_cc1 -fclang-abi-compat=latest -triple amdgcn %s -emit-llvm -o - | FileCheck %s
```

Comment:
Why do you need -fclang-abi-compat=latest

Reply:
Copy/paste from other tests Lol